documentation/plain_rules.md

```diff
@@ -76,7 +76,15 @@ The `type` field is mandatory and must be one of
 * `post`: make an HTTP POST
 * `twitter`: send a twitter
 
-The actions `sms`, `email`, `post` include a field `template` used to build the body of message/request. This text can include placeholders for attributes of the generated event. That placeholder has the form `${` *attribute* `}` which will be substituted by the value of the event's attribute.  The name of the attribute must be prefixed with "ev__" if no alias in the `select` clause is provided. All alias for simple event attributes or "complex" calculated values can be directly used in the placeholder with their name. Any of the original event attributes can be referred as **ev__** *name* (double underscore), if an alias hasn't been given to them.
+
+### Attribute substitution
+
+Some of the fileds of `parameters` form an `action` can include a reference to one of the fields of the notification/event. This allows include information as the pression value received, the id of the device, etc. For example, the actions `sms`, `email`, `post` include a field `template` used to build the body of message/request. This text can include placeholders for attributes of the generated event. That placeholder has the form `${` *attribute* `}` which will be substituted by the value of the event's attribute. All alias for simple event attributes or "complex" calculated values can be directly used in the placeholder with their name. And any of the original event attributes can be referred too.
```

Comment:
I would be more generic here, given that you are providing the explciit list at the end, e.g:

```
Some of the fileds of an `action` (see detail below) of can include...
```

Btw, there is a typo: fileds -> fields